.stickler.yml

```diff
@@ -0,0 +1,3 @@
+linters:
+  rubocop:
+    display_cop_names: true
```

Comment:
#### `REQUIRED`

### Awesome Job adding `.stickler.yml` file. :blush: 

Let's also, add the `.rubocop.yml` file to the root of the project, this can be found [here](https://github.com/microverseinc/linters-config/blob/master/ruby/.rubocop.yml)

> Remember that `.stickler.yml` and `.rubocop.yml` file names start with a **dot**

If you are having an issue with setting up linter, read the `Troubleshooting` section of [this repo](https://github.com/microverseinc/linters-config/tree/master/ruby#troubleshooting).

### You can run the linter locally:  💥 - run `rubocop` in your local env:
>     - add `gem 'rubocop'` to `Gemfile` (not sure how to use Gemfile? Read [this](https://bundler.io/v1.15/guides/bundler_setup.html))
>     - run `bundle install`
>     - copy [.rubocop.yml](https://github.com/microverseinc/linters-config/blob/master/ruby/.rubocop.yml) to the root directory of your project
>     - run `rubocop`
>     - fix linter errors
>     - **Remember to let your Code Reviewer know that you had problems with Stickler and you used linter in local env.**